homeassistant/components/owntracks/messages.py

```diff
@@ -332,10 +332,7 @@ async def async_handle_waypoints_message(hass, context, message):
         if user not in context.waypoint_whitelist:
             return
 
-    if "waypoints" in message:
-        wayps = message["waypoints"]
-    else:
-        wayps = [message]
+    wayps = message.get("waypoints", [message])
```

Comment:
The downside here is that we create a list every time even if it's not needed. Probably not performance sensitive code though.